ts/main.ts

```diff
@@ -0,0 +1,64 @@
+// interface for entry
+interface Entry {
+  entryID: number;
+  title: string;
+  imgUrl: string;
+  notes: string;
+}
+
+// query for DOM elements
+const $form = document.querySelector('form') as HTMLFormElement;
+const $imgPreview = document.querySelector(
+  '.img-placeholder',
+) as HTMLImageElement;
+const $imgUrlInput = document.getElementById('url') as HTMLInputElement;
+const $titleInput = document.getElementById('title') as HTMLInputElement;
+const $notesInput = document.getElementById('notes') as HTMLInputElement;
+
+// check for query errors
+if (!$form) throw new Error('$form query failed');
+if (!$imgPreview) throw new Error('$imgPreview query failed');
+if (!$imgUrlInput) throw new Error('$imgUrlInput query failed');
+if (!$titleInput) throw new Error('$titleInput query failed');
+if (!$notesInput) throw new Error('$notesInput query failed');
+
+// add event listener for img url input
+$imgUrlInput.addEventListener('input', () => {
+  const $newImgUrl = $imgUrlInput.value;
+
+  // set the src attribute
+  $imgPreview.src =
+    $newImgUrl ||
+    'https://img.icons8.com/?size=100&id=qItA4cPLu8rh&format=png&color=c5c5c5';
```

Comment:
If you are going to be doing this, you should be using the placeholder image we provided for you in the `images` folder.